### ISSUE TITLE
/ for division is deprecated

### ISSUE THREAD

**Issue by chocolatkey** [vrimar/construct-ui#65]
When using Dart Sass to compile the SASS, including contruct-ui, I get the following warning:
```
Using / for division is deprecated and will be removed in Dart Sass 2.0.0.

Recommendation: math.div($padding, 2.5)

More info and automated migrator: https://sass-lang.com/d/slash-div

   ╷
25 │       margin:0 0 floor($padding / 2.5) 0;
   │                        ^^^^^^^^^^^^^^
```

It seems `/` is used extensively throughout the project.

**Comment by jdiderik** [comment]
I ran into this issue myself too, using parcel bundler throwing warnings on the divisions.
I went through all scss in my personal fork to fix this.
I may create a pull request for this if needed (and if time permits it), but not sure if this will be done for release 1.0.0 anyway so it is a matter of being patient :-)

**Comment by vrimar** [comment]
Yep, fixed in v0.3.2.
